pkg/apis/imageregistry/v1/types.go

```diff
@@ -203,7 +207,19 @@ type ImageRegistryConfigStorageS3 struct {
 }
 
 type ImageRegistryConfigStorageGCS struct {
+	// Bucket is the bucket name in which you want to store the registry's data
```

Comment:
It might be something new, but we shouldn't use Go names there. Instead we should use JSON names as these comments go to `oc explain`.

Reply:
Yep, must be new, everything else uses the Go names in here.  Should we make changing them all to lowercase a separate pr?

Reply:
@coreydaley yes, we can fix in a separate PR. @dmage called this out in openshift/api as an overall gap in our godocs.